packages/flutter/lib/src/cupertino/text_selection.dart

```diff
@@ -5,6 +5,7 @@
 import 'dart:math' as math;
 
 import 'package:flutter/foundation.dart' show ValueListenable, clampDouble;
+import 'package:flutter/material.dart' show TextSelectionTheme;
```

Comment:
I don't think we have a precedent for importing `material` into the `Cupertino` library (outside of docImports) so i'm not sure if we want to do this. Do you have any thoughts on this @dkwingsmt considering `TextSelectionTheme` is in the material library. I know in the other direction, material importing cupertino, we do this but generally to support the `.adaptive` constructor of certain widgets.

Does wrapping your `TextField` in a `CupertinoTheme` fit your use-case? This is the guidance in https://main-api.flutter.dev/flutter/material/TextSelectionThemeData/selectionHandleColor.html

```
  child: CupertinoTheme(
    data: const CupertinoThemeData(
      primaryColor: myColor,
    ),
    child: TextField(),
  ),
```

Reply:
Huum, it really makes sense, maybe `selectionHandleColor` could be a CupertinoTheme property inherited from MaterialTheme, what do you think?